backend/LexBoxApi/Services/PermissionService.cs

```diff
@@ -94,6 +94,16 @@ public async ValueTask AssertCanViewProject(string projectCode)
         if (!await CanViewProject(projectCode)) throw new UnauthorizedAccessException();
     }
 
+    public async ValueTask<bool> CanViewProjectMembers(Guid projectId)
+    {
+        if (User is not null && User.Role == UserRole.admin) return true;
+        // Project managers can view members of their own projects, even confidential ones
+        if (await CanManageProject(projectId)) return true;
+        var isConfidential = await projectService.LookupProjectConfidentiality(projectId);
+        if (isConfidential is null) return false; // Private by default
```

Comment:
while I agree we want private by default in most cases, this might not actually be one. That said it could encourage users to mark the project type so they can see the list, I'm not sure though.

Reply:
Okay, commit d2719fb6 changes the default for project members to be "visible if confidentiality not specified", and I've verified that it works.